src/ubx.cpp

```diff
@@ -926,7 +926,30 @@ int GPSDriverUBX::configureDevice(const GPSConfig &config, const int32_t uart2_b
 		}
 	}
 
-	if (_mode == UBXMode::RoverWithStaticBaseUart2 || _mode == UBXMode::RoverWithMovingBase) {
+	if (_mode == UBXMode::Normal && _ppk_output) {
+		UBX_DEBUG("Configuring Normal with MSM7 output");
+		cfg_valset_msg_size = initCfgValset();
+
+		// Enable output protocols on UART1
+		cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1OUTPROT_UBX, 1, cfg_valset_msg_size);
+		cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1OUTPROT_RTCM3X, 1, cfg_valset_msg_size);
```

Comment:
This isn't working to enable RTCM output when the module is configured as a Normal Rover. Not sure what's missing...